src/main/java/org/alfresco/rest/api/search/SearchSQLApiWebscript.java

```diff
@@ -80,6 +82,10 @@ public void execute(WebScriptRequest webScriptRequest, WebScriptResponse res) th
             //Turn JSON into a Java object representation
             SearchSQLQuery searchQuery = extractJsonContent(webScriptRequest, assistant.getJsonHelper(), SearchSQLQuery.class);
             SearchParameters sparams = buildSearchParameters(searchQuery);
+            if(searchQuery.getFormat().equalsIgnoreCase("solr"))
+            {
+                sparams.addExtraParameter("format", "solr");
```

Comment:
Not so sure if we should hard code the different formats (e.g. "solr") we have or extract them?